pkg/api/cluster_types.go

```diff
@@ -143,6 +143,7 @@ type Cluster struct {
 	IdentityProviderID string        `json:"identity_provider_id"`
 	ClusterDNS         string        `json:"cluster_dns"`
 	ClientID           string        `json:"client_id"`
+	ClientSecret       string        `json:"client_secret"`
```

Comment:
I am wondering if these two should be named like:
```suggestion
	KasFleetshardClientID           string        `json:"kas_fleetshard_client_id"`
	KasFleetshardClientSecret       string        `json:"kas_fleetshard_client_secret"`
```

I am suggesting that because client_id and client_secret are two generic and also in the long run, I feel like the same approach will need to be done for the cluster identity provider client id and client secret - https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/8ac8f3584c4f86f712a0833449df7976cd3dbf43/internal/kafka/internal/workers/clusters_mgr.go#L1089 so if we keep the client_id, client_secret naming it might be confusing, it might lead to questions like "which client id, or which client secret"
@ziccardi @akoserwal 


this might require a database migration for the `client_id` as it exists already.

Reply:
Sure. We can do it when we start storing the kafka sre client id/secrets. We just need to remember that.